website/server/controllers/api-v3/tasks.js

```diff
@@ -509,11 +509,14 @@ api.scoreTask = {
     if (validationErrors) throw validationErrors;
 
     let user = res.locals.user;
+    let scoreNotes = req.body.scoreNotes;
```

Comment:
can we add some validation limiting how big the note can be? Also we should document the field